flake.nix

```diff
@@ -10,51 +9,50 @@
 
   outputs = { self, cognix }@inputs: (cognix.lib.cognixFlake inputs {}) // {
     packages.x86_64-linux = let
-      lib = cognix.inputs.nixpkgs.lib;
-      callCognix = args: cognix.legacyPackages.x86_64-linux.callCognix {
+      sourceRev = self.sourceInfo.rev or self.sourceInfo.dirtyRev or null;
+      callCognix = args: cognix.legacyPackages.x86_64-linux.callCognix ({ lib, ... }: {
         paths.projectRoot = self;
         imports = [ args ];
-      } "${self}";
+        dockerTools.streamLayeredImage.config = lib.mkIf (sourceRev != null) {
+          Labels."org.opencontainers.image.revision" = sourceRev;
+        };
+      }) "${self}";
       makeRunner = name: architectures: env: callCognix {
         inherit name;
         cog-triton = {
-          runnerOnly = true;
           inherit architectures;
+          # only grab deps of nvidia-pytriton, transformers
+          rootDependencies = [ "nvidia-pytriton" "transformers" ];
         };
         cognix.environment = env;
-        dockerTools.streamLayeredImage.config = lib.mkIf (self.sourceInfo ? rev || self.sourceInfo ? dirtyRev) {
-          Labels."org.opencontainers.image.revision" = self.sourceInfo.rev or self.sourceInfo.dirtyRev;
-        };
       };
-    in {
-      default = self.packages.x86_64-linux.cog-triton-mistral-7b;
-      cog-triton-builder = callCognix ({config, pkgs, ... }: {
-        name = "cog-triton-builder";
-        # only grab deps of cog, tensorrt-llm
-        python-env.pip.rootDependencies = lib.mkForce (lib.genAttrs [
-          "cog" "tensorrt-llm" "omegaconf" "hf-transfer"
-        ] (x: true));
-        # override cog.yaml:
-        cog = {
-          concurrency = lib.mkForce 1;
+      makeBuilder = name: callCognix ( { config, lib, ... }: {
+        inherit name;
+        cog-triton = {
+          # only grab deps of tensorrt-llm, omegaconf, hf-transfer
+          rootDependencies = [ "tensorrt-llm" "omegaconf" "hf-transfer" ];
```

Comment:
if I'm reading this right and it's only the runner image I don't think it should need hf-transfer